gamemode/config/hunger.lua

```diff
@@ -0,0 +1,13 @@
+GM.Config = GM.Config or  {}
+
+GM.Config.Hunger = {}
+local Config = GM.Config.Hunger
+
+Config.MAX_FOOD = 100
+Config.MAX_WATER = 100
+Config.DELAY = 0.5
+Config.LOSS_CHANCE = 0.1 -- 0 - 1 
```

Comment:
10% Chance to lose hunger?

Reply:
yeah, we might want to lower that or raise the delay eventually